profiles/profiles.desc

```diff
@@ -336,6 +336,7 @@ amd64		default/linux/amd64/17.0/musl/clang			exp
 amd64		default/linux/amd64/17.0/musl/hardened			exp
 amd64		default/linux/amd64/17.0/musl/hardened/selinux		exp
 arm		default/linux/arm/17.0/musl/armv6j			dev
+arm             default/linux/arm/17.0/musl/armv6j/llvm                 dev
```

Comment:
exp please (because none of the other LLVM profiles are dev yet iirc)

Reply:
This change has now been made